File: help/eng_gb/exercicios_da_ficha_6/Ficha06_002/Ficha06_002.cpp
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <iostream>
#include <cstring>
#include <cmath>
using namespace std;

// Class definition
class Ficha06_002 {

    public:
        int main() {
            cout << "Height : ";
            int height;
            cin >> height;
            int i  = 0 ;
            for( int i = 0; i < height; i = i + 1 ) {
                int j  = 0 ;
                for( int j = 0; j < height; j = j + 1 ) {
                    cout << "*";
                }
                cout << "\n";
            }
            return 0;
        }
};// end of class Ficha06_002
// Execute main function
int main(){
    Ficha06_002 application; // create object
    application.main(); // execute object
    return 0;
}

//C++  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Fri, 2022 Sep 30 21:55

File: help/eng_gb/exercicios_da_ficha_6/Ficha06_002/Ficha06_002.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha06_002 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Height : " );
        int height = Integer.valueOf(keyb.nextLine());
        for( int i = 0; i < height; i = i + 1 ) {
            for( int j = 0; j < height; j = j + 1 ) {
                System.out.print("*" );
            }
            System.out.print("\n" );
        }
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Fri, 2022 Sep 30 21:55
